install_scripts/install_udp_rx.sh

```diff
@@ -0,0 +1,72 @@
+# create a service user
+echo "Creating User"
+useradd -r udprx
+# create /etc/udp_rx directory
+echo "Creating udp_rx directories"
+mkdir -p /etc/udp_rx
+chmod 755 /etc/udp_rx
+# move the default portlist to that directory
+echo "Moving portslist and setting permissions"
+cp portslist /etc/udp_rx/
+chown root /etc/udp_rx/portslist
+chgrp root /etc/udp_rx/portslist
+chmod 644 /etc/udp_rx/portslist
+# move the default config file to that directory
+echo "Moving configuration files"
+cp udp_rx_conf.json /etc/udp_rx/
+chown root /etc/udp_rx/udp_rx_conf.json
+chgrp root /etc/udp_rx/udp_rx_conf.json
+chmod 644 /etc/udp_rx/udp_rx_conf.json
+# if the keys and the CA cert exist, move them
+# key 
+if [ -e udp_rx.key ]
```

Comment:
The install script checks for everything to exist. Does this mean we install udp_rx on the vhd and do all of our testing, then delete the ca we were testing with before release? I think that's fine I just want to be sure I understand.

Reply:
udp_rx installer doesn't necessarily care about the keys existing. This will just move them if they happen to be there. We can do whatever we want for dev->prod release